loaders/apirefloader.fsx

```diff
@@ -30,10 +30,10 @@ let loader (projectRoot: string) (siteContent: SiteContents) =
     try
         let dir = Path.Combine(projectRoot, "packages", "FSharp.Core", "lib", "netstandard2.0")
         let xml = Path.Combine(dir, "FSharp.Core.xml")
-        let dll = Path.Combine(System.AppContext.BaseDirectory, "FSharp.Core.dll")
+        let dll = [ Path.Combine(System.AppContext.BaseDirectory, "FSharp.Core.dll") ]
```

Comment:
maybe to change the name from dll to dlls (since the generator supports a sequence now)?